src/olympia/lib/settings_base.py

```diff
@@ -1665,6 +1665,10 @@ def get_redis_settings(uri):
 
     # We don't allow refreshes, instead we simply have a long duration.
     'JWT_ALLOW_REFRESH': False,
+
+    # Prefix for non-apikey jwt tokens. Should be different from 'JWT' which we
+    # already used for api key tokens.
+    'JWT_AUTH_HEADER_PREFIX': 'JWT-i',
```

Comment:
That sounds good to me.

I feel like we should be able to write an authentication class that decodes the JWT and delegates to the right version based on the payload's type. This is definitely simpler though. When we support authentication for external clients it would be nice for them to user the `Bearer` scheme as well. I'd imagine they could use the same JWTs we just need a way for them to get one.